docs/product-getting-started-guide/src/main/asciidoc/task_data-model.adoc

```diff
@@ -1,66 +1,157 @@
 [id='_defining_a_data_model']
-= Defining a Data Model
```

Comment:
I think we shouldn't be talking about data objects in terms of data models anymore. Central doesn't have any "Data Model" assets when you go to create stuff, for instance, nor is there much (if any) reference to it in the UI, from what I can tell. So talking about data objects in terms of their collection in data models _might_ be helpful conceptually, but I don't think so, imo. If it were me, I'd just make this a chapter on creating the necessary data objects (two), since that's the bottom line anyway, from the user's perspective. This is what I've done in my [GDT doc](http://file.rdu.redhat.com/~sterobin/BXMSDOC-2008/#data-objects-con) for example. 

Something like:
**2. Creating the traffic violation data objects** <concept module>
<What data objects are, how/why they're necessary for this tutorial, etc. See my [GDT doc](http://file.rdu.redhat.com/~sterobin/BXMSDOC-2008/#data-objects-con) for example. I use the same Data Objects section in my guided rules doc as well (coming soon). Ideally you should just reuse that exact same concept module file that I used, and then add any unique text after it that you may need for this assembly in your main.adoc file, following the include link.>
**2.1 Creating the Violation data object** <Procedure module>
<Context statement - why this data object is important and/or what it's used for>
**.Procedure**
<steps>
**2.1.1 Adding the Violation data object fields** <procedure module>
<Context statement - what the fields are used for>
**.Procedure**
<steps>
**2.1 Creating the Driver data object** <Procedure module>
<Context statement - why this data object is important and/or what it's used for>
**.Procedure**
<steps>
**2.1.1 Adding the Driver data object fields** <procedure module>
<Context statement - what the fields are used for>
**.Procedure**
<steps>

Reply:
This makes good sense Stetson. I have included a draft message for Ivo in my SME review request. Additionally, I think reuse regarding data objects if the right way to go. I will give that a try and will shout out if I have any issue with the proper linking.
Thank you for the great feedback :)